src/modules/traceModelContext.js

```diff
@@ -0,0 +1,99 @@
+import { SessionManager } from './sessionManager.js';
+
+class TraceModelContext {
```

Comment:
No class documentation?

Reply:
I think the documentation of the constructor is sufficient